package.json

```diff
@@ -94,10 +94,11 @@
   "scripts": {
     "benchmark-rule": "node scripts/benchmark-rule.js",
     "flow": "flow",
-    "jest": "jest",
+    "jest": "jest --coverage",
```

Comment:
Even though this PR is currently passing the Travis CI tests I think this is going to be an issue per #2174 

_FYI: Once I've finished reviewing this PR I'll take another look at #2174 and add another note here_